src/components/Form/ErrorMessage/ErrorMessage.js

```diff
@@ -25,7 +25,7 @@ export const ErrorMessage = ({ error, success, ErrorComponent, SuccessComponent
   if (success && SuccessComponent) return SuccessComponent
   return (
     <div>
-      <Announced aria-live="polite" message={error || undefined} />
+      <Announced aria-live="polite" message={'Error: ' + error || undefined} />
```

Comment:
`'Error: ' + error || undefined` will never be able to evaluate to undefined here.

For reference (I had to look this up to figure it out)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence